maestro-driver-ios/maestro-driver-iosUITests/Routes/Models/SwipeRequest.swift

```diff
@@ -1,7 +1,31 @@
-struct SwipeRequest: Codable {
-    let startX: Float
-    let startY: Float
-    let endX: Float
-    let endY: Float
-    let velocity: Float?
+import Foundation
+
+struct SwipeRequest: Decodable {
+
+    enum CodingKeys: String, CodingKey {
+        case startX, startY, endX, endY, duration
+    }
+
+    let start: CGPoint
+    let end: CGPoint
+    let duration: Double
```

Comment:
may be we could call it durationMs for better readability

Reply:
These are seconds, the units of iOS, just like Double (CGFloat) for pixel locations

Reply:
may be could use TimeInterval for clarity